app/src/componentLibrary/attributeList/editableAttribute/attribute/attribute.jsx

```diff
@@ -45,7 +45,7 @@ export const Attribute = ({
   return (
     <div
       ref={wrapperRefCallback}
-      tabIndex={disabled ? -1 : 0}
+      tabIndex={0}
```

Comment:
What is the reason for removing `disabled ? -1 : 0`

If the attr is disabled want to be able to focus on this attr.

Reply:
according to design